src/main/fc/rc_modes.c

```diff
@@ -41,7 +41,7 @@ static bool isUsingNAVModes = false;
 #endif
 
 boxBitmask_t rcModeActivationMask; // one bit per mode defined in boxId_e
-STATIC_ASSERT(CHECKBOX_ITEM_COUNT <= 32, too_many_box_modes);
+STATIC_ASSERT(CHECKBOX_ITEM_COUNT <= 33, too_many_box_modes);
```

Comment:
I think this is not needed anymore since you did not added a new BOX but reused LLIGHTS

Reply:
You're right.